src/Exceptions/RLSCommentConstraintException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Exceptions;
```

Comment:
Move to `src/RLS/Exceptions`

Reply:
Moved https://github.com/archtechx/tenancy/pull/1354/commits/8934114f92789265559372f3e9b3bae60982ecfc